assets/app/scripts/filters/resources.js

```diff
@@ -412,6 +412,28 @@ angular.module('openshiftConsole')
       return details.id;
     };
   })
+  .filter('isRecentBuild', function(ageLessThanFilter) {
```

Comment:
No, but I couldn't come up with better. Maybe just move this code to overview controller and handle it there if it's so specific to the overview page.